conda_build/utils.py

```diff
@@ -1234,6 +1234,7 @@ def apply_pin_expressions(version, min_pin='x.x.x.x.x.x.x', max_pin='x'):
 def filter_files(files_list, prefix, filter_patterns=('(.*[\\\\/])?\.git[\\\\/].*',
                                                       '(.*[\\\\/])?\.git$',
                                                       '(.*)?\.DS_Store.*',
+                                                      '.*\.la',
```

Comment:
PS: last time I wrote a regular expression was many years ago! So I'm not certain this is bullet proof. (It work on a few feedstocks I tested.)

Reply:
I think this regex needs a trailing $ so it doesn't match files like `example.lab`